pkg/reconciler/instances/ory/ory.go

```diff
@@ -18,10 +18,13 @@ func init() {
 	}
 
 	reconciler.
-		WithPreReconcileAction(&preAction{
+		WithPreReconcileAction(&preInstallAction{
```

Comment:
The name change implies "installation" operation. Will the same code also handle upgrades? If so, then the new name isn't really good.

Reply:
That's a good point. It is used for upgrades as well. I will rename it to preReconcileAction and postReconcileAction respectively.